scripts/data/common/killed-in-gaza/reconcile_lists.ts

```diff
@@ -980,15 +981,15 @@ async function reconcileCSVs(
   console.log("new list length:", newListLength);
   console.log("new list community records:", communityRecords.size);
   if (process.argv.includes("--write")) {
-    fs.writeFileSync(
+    fs.appendFileSync(
```

Comment:
it should overwrite the file (not append) since the list isn't all new names, there will be duplicates and that's what this script is doing: reconciling existing ones by ID and updating existing records as needed while appending any new ones where there's no conflict